inference-engine/tests/unit/inference_engine/cpp_interfaces/ie_memory_state_internal_test.cpp

```diff
@@ -27,7 +27,7 @@ class InferRequestVariableStateTests : public ::testing::Test {
     shared_ptr<MockIVariableStateInternal> mockVariableStateInternal;
     MockIInferencePlugin*                           mockIPlugin;
     InferencePlugin                                 plugin;
-    SoExecutableNetworkInternal                     net;
+    details::SOPointer<IExecutableNetworkInternal>  net;
```

Comment:
can we remove `SOPointer` from here as well?

Reply:
done